README.md

```diff
@@ -7,6 +7,7 @@ Current features:
 -underground and surface lakes and lava lakes  
 -lava lakes can be turned on and off  
 -lakes will create a small cave above them  
+-desert lakes are less probable and will turn into palm tree oasis
```

Comment:
Fun fact: "oasis" has a plural version "oases" (i had leaned toward "oasi" but looks like it is "oases")